src/api/tracking/withdrawals.ts

```diff
@@ -0,0 +1,106 @@
+import axios from '../';
+import { addError } from '../utils';
+import { paths } from 'src/api/types/generated-schema';
+import {
+  SearchCriteria,
+  SearchRequestPayload,
+  SearchResponseElement,
+  convertToSearchNodePayload,
+  search,
+} from 'src/api/search';
+import { Batch, NurseryWithdrawal } from 'src/api/types/batch';
+import { Delivery } from 'src/api/types/tracking';
+
+/**
+ * List nursery withdrawals
+ */
+export async function listNurseryWithdrawals(
+  organizationId: number,
+  searchCriteria: SearchCriteria,
+  count = 1000
+): Promise<SearchResponseElement[] | null> {
+  const searchParams: SearchRequestPayload = {
+    prefix: 'nurseryWithdrawals',
+    fields: [
+      'id',
+      'delivery_id',
+      'withdrawnDate',
+      'purpose',
+      'facility_name',
+      'destinationName',
+      'plotNames',
+      'batchWithdrawals.batch_species_scientificName',
```

Comment:
Change this to `batchWithdrawals.batch.species_scientificName` and you won't have to do the deduping later.

Reply:
That's a period rather than an underscore after `batch`.

Reply:
Oh wow, awesome, let me try that. Thanks!!!

Reply:
In a search field path, you can think of `.` as, "Return a list of child objects" and `_` as, "Return a separate copy of the parent object for each child value."

Reply:
Oh, wait, hang on, I misread what the deduping code was doing. I think you still need it and the field name you have here is the right one. Sorry for the false alarm.

Reply:
Oh no worries, thanks for looking at the review, really appreciate that.